### ISSUE TITLE
Github hook calling "refresh" throws exception

### ISSUE THREAD

**Issue by rbkreisberg** [GitbookIO/nuts#59]
It looks like the GitHubBackend class does not have a *versions* member.

I activated the hook from github.  Tailing the heroku logs I see:

```
2016-04-14T20:25:05.963441+00:00 app[web.1]: /app/lib/backends/github.js:42
2016-04-14T20:25:05.963476+00:00 app[web.1]:         that.versions.list();
2016-04-14T20:25:05.963479+00:00 app[web.1]:                      ^
2016-04-14T20:25:05.963480+00:00 app[web.1]:
2016-04-14T20:25:05.963481+00:00 app[web.1]: TypeError: Cannot read property 'list' of undefined
2016-04-14T20:25:05.963482+00:00 app[web.1]:     at EventEmitter.<anonymous> (/app/lib/backends/github.js:42:22)
2016-04-14T20:25:05.963482+00:00 app[web.1]:     at emitOne (events.js:90:13)
2016-04-14T20:25:05.963483+00:00 app[web.1]:     at EventEmitter.emit (events.js:182:7)
2016-04-14T20:25:05.963484+00:00 app[web.1]:     at BufferList._callback (/app/node_modules/github-webhook-handler/github-webhook-handler.js:84:15)
2016-04-14T20:25:05.963484+00:00 app[web.1]:     at BufferList.end (/app/node_modules/bl/bl.js:83:10)
2016-04-14T20:25:05.963485+00:00 app[web.1]:     at IncomingMessage.onend (_stream_readable.js:498:10)
2016-04-14T20:25:05.963486+00:00 app[web.1]:     at IncomingMessage.g (events.js:273:16)
2016-04-14T20:25:05.963486+00:00 app[web.1]:     at emitNone (events.js:80:13)
2016-04-14T20:25:05.963487+00:00 app[web.1]:     at IncomingMessage.emit (events.js:179:7)
2016-04-14T20:25:05.963493+00:00 app[web.1]:     at endReadableNT (_stream_readable.js:913:12)
2016-04-14T20:25:06.986430+00:00 heroku[web.1]: Process exited with status 1
2016-04-14T20:25:07.003944+00:00 heroku[web.1]: State changed from up to crashed
```

**Comment by djpereira** [comment]
I see exactly the same issue.

**Comment by rbkreisberg** [comment]
fixed by #63
